### ISSUE TITLE
Make GitHub Actions for another stack

### ISSUE THREAD

**Issue by jenweber** [cardstack/cardstack#1393]
When we switched to GitHub Actions instead of Travis, we didn't set up both demo and prod.

**Comment by jenweber** [comment]
Note to self, the git repo keys are replaced by using a github https.

**Comment by jenweber** [comment]
It looks like the deploy job didn't run. I must have something incorrect in the logic.

<img width="1439" alt="Screen Shot 2020-03-02 at 5 01 07 PM" src="https://user-images.githubusercontent.com/16627268/75722055-732f6080-5ca7-11ea-9179-f4e073abbd14.png">

**Comment by jenweber** [comment]
Need to reopen https://github.com/cardstack/cardstack/pull/1397 and push latest changes to `production` branch

**Comment by jenweber** [comment]
The conditional is fixed. Next up is this error:

```
 + remote=unix:///tmp/cardstack-remote-docker-demo
+ docker -H unix:///tmp/cardstack-remote-docker-demo stack deploy --with-registry-auth -c ./deploy/docker-compose.yml hub
+ socat UNIX-LISTEN:/tmp/cardstack-remote-docker-demo,reuseaddr,fork 'EXEC:ssh -i /home/runner/.ssh/id_rsa -T docker-control@'
ssh: Could not resolve hostname : Name or service not known
```

**Comment by jenweber** [comment]
Closed by #1397
